src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/QueryHandlers/CreateVenueHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Models;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+using Dfc.CourseDirectory.Core.Models;
+using Microsoft.Azure.Documents.Client;
+using OneOf.Types;
+
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.QueryHandlers
+{
+    public class CreateVenueHandler : ICosmosDbQueryHandler<CreateVenue, Success>
+    {
+        private readonly SqlDataSync _sqlDataSync;
+
+        public CreateVenueHandler(SqlDataSync sqlDataSync)
+        {
+            _sqlDataSync = sqlDataSync;
+        }
+
+        public async Task<Success> Execute(
+            DocumentClient client,
+            Configuration configuration,
+            CreateVenue request)
+        {
+            var documentUri = UriFactory.CreateDocumentUri(
+                configuration.DatabaseId,
+                configuration.VenuesCollectionName,
+                request.VenueId.ToString());
+
+            var venue = new Venue()
+            {
+                VenueName = request.Name,
+                Email = request.Email,
+                PHONE = request.PhoneNumber,
+                Website = request.Website,
+                AddressLine1 = request.AddressLine1,
+                AddressLine2 = request.AddressLine2,
+                Town = request.Town,
+                County = request.County,
+                Postcode = request.Postcode,
+                Latitude = request.Latitude,
+                Longitude = request.Longitude,
+                CreatedBy = request.CreatedBy.Email,
+                CreatedDate = request.CreatedDate,
+                UpdatedBy = request.CreatedBy.Email,
```

Comment:
I thought we were using `UserId` for this moving forward?

Reply:
That hasn't been changed for any other Venue journeys yet as far as I know and I wanted to keep consistent.

Reply:
I thought Tim did a PR to update them all the other week? Either way, moving forward we should be using the `UserId`, so using `Email` here is incorrect. And if there's others in other journeys that need changing we either need to get it done or put a ticket in the backlog to do it.

Reply:
tldr; Correctness trumps consistency.

Reply:
>tldr; Correctness trumps consistency.
I disagree.

Having multiple approaches for a given entity is a mistake, I feel.

Reply:
I've just looked at the code, including the other Venues journey - everything else is now using `UserId`, not `Email`. Please change this to `UserId`.

Reply:
Doh should've rebased before checking other journeys. Changing...